be/src/exec/vectorized/sorting/sort_column.cpp

```diff
@@ -458,6 +458,14 @@ Status sort_vertical_chunks(const bool& cancel, const std::vector<Columns>& vert
 }
 
 void append_by_permutation(Chunk* dst, const std::vector<ChunkPtr>& chunks, const Permutation& perm) {
+    std::vector<const Chunk*> src;
+    for (auto& chunk : chunks) {
```

Comment:
how many chunks at most here?

Reply:
no limitation

Reply:
if no limit, I think we should reserve the vector.

Reply:
good advice